### PR TITLE
Added LowQ2 plugin with hit digitization

### PR DESCRIPTION
**Pull request by simonge**
### Briefly, what does this PR introduce?
Adds a basic LowQ2 plugin which only digitizes the simulated hits using the standard silicon digitization factory

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
no

### Does this PR change default behavior?
Adds digitized lowq2 tagger hits to output

### CODE REVIEW

File: src/detectors/LOWQ2/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+plugin_glob_all(${PLUGIN_NAME})
+
+# Find dependencies
+plugin_add_dd4hep(${PLUGIN_NAME})
+plugin_add_event_model(${PLUGIN_NAME})
+
+# Add include directories (works same as target_include_directories)
+plugin_include_directories(${PLUGIN_NAME} SYSTEM PUBLIC ${podio_INCLUDE_DIR} ${EDM4EIC_INCLUDE_DIR})
+
+# Add libraries (works same as target_include_directories)
```
Comment (wdconinc) [inline src/detectors/LOWQ2/CMakeLists.txt:25]:
```suggestion
```

File: src/detectors/LOWQ2/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.16)
+
```
Comment (wdconinc) [inline src/detectors/LOWQ2/CMakeLists.txt:2]:
```suggestion
```

**Comment by wdconinc** [comment]
Two minor comments, then can be approved and merged.

File: src/detectors/LOWQ2/CMakeLists.txt (hunk 2 above)
Comment (simonge) [reply src/detectors/LOWQ2/CMakeLists.txt]: Should be done now

**Review by wdconinc — APPROVED** [review]

**Pull request merged**